### ISSUE TITLE
Make it easier to see updated items in order manager

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/figgy#6000]
### Summary

When selecting an image in order manager, there is a very narrow orange line around the image. Can this be thicker, or a different color? When the image's label has been changed, the image's box turns orange making it extremely difficult to determine which image is currently selected. 

Request from Joyce.

### Impact 

Accessibility improvement

### Priority recommendation

- [ ] asap
- [ ] within the next 3 weeks
- [x] PO will prioritize

**Comment by hackartisan** [comment]
Joyce said recently on slack that this has improved, maybe a browser setting?

**Comment by kelea99** [comment]
I noticed it too, @hackartisan . it is almost like its glowing.  See Screenshot:
<img width="1512" alt="Screenshot 2024-04-05 at 3 35 06 PM" src="https://github.com/pulibrary/figgy/assets/45043014/60c00f95-c6d0-4192-8a84-a36d6c6cf925">
